tests/zfs-tests/include/libtest.shlib

```diff
@@ -34,6 +34,8 @@
 . ${STF_SUITE}/include/math.shlib
 . ${STF_SUITE}/include/blkdev.shlib
 
+. ${STF_SUITE}/include/tunables.cfg
+
```

Comment:
Can you add a comment above `get_tunable`, `set_tunable` wrappers referencing the new ` tunables.cfg` file and explaining that these functions must be called using the `VARIABLE` names from the config file.  It would be nice to include a little more explanation at the top of `tunables.cfg` regarding the purpose of the file.